src/pyubx2/ubxreader.py

```diff
@@ -230,6 +234,7 @@ def _parse_nmea(self, hdr: bytes) -> tuple:
         Parse remainder of NMEA message (using pynmeagps library).
 
         :param bytes hdr: NMEA header ($G or $P)
+        :param bool parsing: a boolean flag to determine if the stream needs to be parsed

```

Comment:
can remove this docstring entry as it no longer applies

Reply:
Removed